lib/api/src/js/externals/function.rs

```diff
@@ -607,7 +608,7 @@ mod inner {
     use std::marker::PhantomData;
     use std::panic::{self, AssertUnwindSafe};
 
-    use wasmer_types::{FunctionType, NativeWasmType, Type};
+    use wasmer_types::{FunctionType, RawValue, NativeWasmType, Type};
```

Comment:
RawValue needs to be `f64` in js. Is that the case?

Reply:
No, fixed now.